src/components/Tooltip/Tooltip.stories.tsx

```diff
@@ -3,18 +3,139 @@
 import React from "react";
 import { Meta, Story } from "@storybook/react";
 import { Tooltip as TooltipComponent, TooltipProps } from "./Tooltip";
+import { BrightHeaderVariants } from "./partials/BrightHeader";
+import IconIcons from "@elements/Icon/Generated/IconIcons";
 
 export default {
     title: "Components/Tooltip",
     component: TooltipComponent,
     args: {
-        tooltip: "Tooltip Content (Reactnode)",
+        tooltip: "Your text here",
+        heading: "",
+        buttons: [],
```

Comment:
I'd remove the default value for `heading` and `buttons` as you've got all possibilities nicely covered in the different stories.